### PR TITLE
Add new attributes

### PR DESCRIPTION
**Pull request by victorvarza**
## Description

Add new attributes for:
- `cloudProviderRegion` - the actual region of the cloud provider
- `extra::ExtendedRegion` - extended internal region which might be used by some cluster-registry consumers
- `managingOrg` - the specific Organization which operates the cluster

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by aalexandru — APPROVED** [review]

File: pkg/apiserver/docs/docs.go [hunk 1]
```diff
@@ -158,11 +161,21 @@ var doc = `{
                     }
                 },
                 "itemsCount": {
+                    "description": "TODO: should be rename to total",
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/docs.go:164]: nit...`renamed`

File: pkg/apiserver/docs/docs.go [hunk 2]
```diff
@@ -172,16 +185,17 @@ var doc = `{
...
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/docs.go:188]: nit...`renamed`

File: pkg/apiserver/docs/swagger.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
                         "bearerAuth": []
                     }
                 ],
-                "description": "List all clusters. Use query parameters to filter results. Auth is required",
+                "description": "List all clusters. Use query parametricsers to filter results. Auth is required",
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/swagger.json:23]: `parametricsers` ... please check again

File: pkg/apiserver/docs/swagger.json [hunk 4]
```diff
@@ -148,11 +151,21 @@
                     }
                 },
                 "itemsCount": {
+                    "description": "TODO: should be rename to total",
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/swagger.json:154]: nit ... rename`d`

File: pkg/apiserver/docs/swagger.json [hunk 5]
```diff
@@ -162,16 +175,17 @@
...
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/swagger.json:178]: nit ... rename`d`

File: pkg/apiserver/docs/docs.go [hunk 6]
```diff
@@ -30,7 +30,7 @@ var doc = `{
                         "bearerAuth": []
                     }
                 ],
-                "description": "List all clusters. Use query parameters to filter results. Auth is required",
+                "description": "List all clusters. Use query parametricsers to filter results. Auth is required",
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/docs.go:33]: `parametricsers` ... typo

File: pkg/apiserver/docs/swagger.yaml [hunk 7]
```diff
@@ -309,8 +336,8 @@ paths:
     get:
       consumes:
       - application/json
-      description: List all clusters. Use query parameters to filter results. Auth
-        is required
+      description: List all clusters. Use query parametricsers to filter results.
```
Comment (JonnieDoe) [inline pkg/apiserver/docs/swagger.yaml:339]: Are swagger.`{json|yaml}` generated from `docs.go`? If so, please correct the issue there and re-generate these files too.

File: pkg/client/webhook/server.go [hunk 8]
```diff
@@ -72,25 +73,29 @@ func (s *Server) webhookHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *Server) process(alert Alert) error {
+
+	// DeadMansSwitchAlert should always firing
```
Comment (JonnieDoe) [inline pkg/client/webhook/server.go:77]: Maybe...DeadMansSwitchAlert should always`fire`?

File: pkg/client/webhook/server.go [hunk 9]
```diff
@@ -72,25 +73,29 @@ func (s *Server) webhookHandler(w http.ResponseWriter, r *http.Request) {
...
 		s.Metrics.RecordDMSLastTimestamp()
 		s.Log.Info("received deadmansswitch", "alertname", DeadMansSwitchAlertName)
 		return nil
 	}
 
-	for _, i := range s.AlertMap {
-		if i.AlertName != alert.CommonLabels.Alertname {
+	for _, a := range s.AlertMap {
+
+		// accept only alerts pre-configured
```
Comment (JonnieDoe) [inline pkg/client/webhook/server.go:86]: `accept only pre-configured alerts`

File: docs/developer-guides/annotations.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+Annotation keys and values can only be strings.
+
+|Name                       | type |
+|---------------------------|------|
+|[clusters.registry.ethos.adobe.com/excluded-tags](#excluded-tags)|string|
+|[clusters.registry.ethos.adobe.com/skip-ca-cert](#skip-ca-cert)|"true" or "false"|
+
+## Excluded tags
+
+The annotation `clusters.registry.ethos.adobe.com/excluded-tags` defines the behavior of the cluster-registry-client regarding to dynamic tags. If Alertmanager sends a signal for a specific tag, it will be ignore and value from `tags` remains unchanged.
```
Comment (JonnieDoe) [inline docs/developer-guides/annotations.md:14]: `ignored`

**Review by JonnieDoe — CHANGES_REQUESTED** [review]

File: pkg/client/webhook/server.go (hunk 9 above)
Comment (victorvarza) [reply pkg/client/webhook/server.go:86]: yup, this is by design

**Pull request merged**